src/custom/constants/tokens/index.ts

```diff
@@ -32,21 +32,20 @@ export const ADDRESS_IMAGE_OVERRIDE = {
 }
 
 export const V_COW: Record<number, Token> = {
-  // TODO: enable once contract addresses are added
-  // [SupportedChainId.MAINNET]: new Token(
-  //   SupportedChainId.MAINNET,
-  //   V_COW_CONTRACT_ADDRESS[SupportedChainId.MAINNET] || '',
-  //   18,
-  //   'vCOW',
-  //   'Virtual CowSwap Token'
-  // ),
-  // [SupportedChainId.XDAI]: new Token(
-  //   SupportedChainId.XDAI,
-  //   V_COW_CONTRACT_ADDRESS[SupportedChainId.XDAI] || '',
-  //   18,
-  //   'vCOW',
-  //   'Virtual CowSwap Token'
-  // ),
+  [SupportedChainId.MAINNET]: new Token(
+    SupportedChainId.MAINNET,
+    V_COW_CONTRACT_ADDRESS[SupportedChainId.MAINNET] || '',
+    18,
+    'vCOW',
+    'Virtual CowSwap Token'
```

Comment:
Not sure how these are used, however the actual symbol for all three deployments is `TESTvCOW` and the name is `*TEST* CoW Protocol Virtual Token`.

Even if it's only used in the user interface, the final token name has changed and is now `CoW Protocol Virtual Token`. You could also import the values here from the npm package, it [exports this information](https://github.com/gnosis/cow-token/blob/main/src/ts/token.ts) and it's kept up to date with a test. (However there is no package version for the test deployment.)

Reply:
Good points in the name.

> however the actual symbol for all three deployments is TESTvCOW

Yes, although in this case is not too relevant. We don't use much (we are mostly interested in the address). If we change it, then we need to change it back, and won't add too much value.

> Even if it's only used in the user interface, the final token name has changed and is now CoW Protocol Virtual Token.

Changed! 

> You could also import the values here from the npm package, it exports this information and it's kept up to date with a test. (However there is no package version for the test deployment.)

For now, we are trying to keep dependencies smaller, and since the coupling between the frontend and the token is so small (just one address per network), it was not worth it to add an additional dependency for us.